packages/neo-one-smart-contract-lib/src/__data__/contracts/TestTransferableContract.ts

```diff
@@ -0,0 +1,27 @@
+import { Address, SmartContract } from '@neo-one/smart-contract';
+// tslint:disable-next-line no-implicit-dependencies
+import { NEP5Token } from '@neo-one/smart-contract-lib';
+import { withTransferableOwnership } from '../../ownership/withTransferableOwnership';
+
+export class TransferableContract extends withTransferableOwnership(NEP5Token(SmartContract)) {
```

Comment:
Remove `NEP5Token` and related properties.

Reply:
👍